src/integrations/prefect-dbt/prefect_dbt/cli/tasks.py

```diff
@@ -69,7 +73,326 @@ def dbt_test_flow():
+        artifact_key=artifact_key,
+        logger=logger,
+    )
+
+    return results
+
+
+@task
+def dbt_seed_task(
+    profiles_dir: Optional[Union[Path, str]] = None,
+    project_dir: Optional[Union[Path, str]] = None,
+    overwrite_profiles: bool = False,
+    dbt_cli_profile: Optional[DbtCliProfile] = None,
+    dbt_client: Optional[dbtRunner] = None,
+    create_artifact: bool = True,
+    artifact_key: str = "dbt-seed-task-summary",
+):
+    """
+    Executes the 'dbt seed' command within a Prefect task,
+    and optionally creates a Prefect artifact summarizing the dbt build results.
+
+    Args:
+        profiles_dir: The directory to search for the profiles.yml file. Setting this
+            appends the `--profiles-dir` option to the command provided.
+            If this is not set, will try using the DBT_PROFILES_DIR env variable,
+            but if that's also not set, will use the default directory `$HOME/.dbt/`.
+        project_dir: The directory to search for the dbt_project.yml file.
+            Default is the current working directory and its parents.
+        overwrite_profiles: Whether the existing profiles.yml file under profiles_dir
+            should be overwritten with a new profile.
+        dbt_cli_profile: Profiles class containing the profile written to profiles.yml.
+            Note! This is optional and will raise an error
+            if profiles.yml already exists under profile_dir
+            and overwrite_profiles is set to False.
+        dbt_client: An instance of a dbtRunner client to execute dbt commands. If None,
+            a new instance is created.
+        create_artifact: If True, creates a Prefect artifact on the task run
+            with the dbt build results using the specified artifact key.
+            Defaults to True.
+        artifact_key: The key under which to store
+            the dbt build results artifact in Prefect.
+            Defaults to 'dbt-seed-task-summary'.
+
+    Example:
+    ```python
+        from prefect import flow
+        from prefect_dbt.cli.tasks import dbt_seed_task
+
+        @flow
+        def dbt_test_flow():
+            dbt_seed_task(
+                project_dir="/Users/test/my_dbt_project_dir"
+            )
+    ```
+
+    Raises:
+        ValueError: If required dbt_cli_profile is not provided
+                    when needed for profile writing.
+        RuntimeError: If the dbt build fails for any reason,
+                    it will be indicated by the exception raised.
+    """
+    logger = get_run_logger()
+    logger.info("Running dbt seed task.")
+
+    results = dbt_run(
+        command="seed",
+        profiles_dir=profiles_dir,
+        project_dir=project_dir,
+        overwrite_profiles=overwrite_profiles,
+        dbt_cli_profile=dbt_cli_profile,
+        dbt_client=dbt_client,
+        create_artifact=create_artifact,
+        artifact_key=artifact_key,
+        logger=logger,
+    )
+
+    return results
+
+
+def dbt_run(
```

Comment:
This `dbt_run` command appears to duplicate the existing `trigger_dbt_cli_command` function in this library. I'd prefer it if we reuse existing functionality in this library when building out this new interfaces. If the existing functionality doesn't support something that's needed for the new tasks and flows, let's chat and figure out a way to update the existing functionality to make it work for you. I want to prevent multiple implementations that will make it harder for users to navigate this library.